src/components/v2/Button/styles.tsx

```diff
@@ -0,0 +1,41 @@
+import { css } from '@emotion/react';
+import { useTheme } from '@mui/material';
+
+export const styles = () => {
+  const theme = useTheme();
+  return css`
+    padding: ${theme.spacing(1.5)} ${theme.spacing(3)};
+    border-radius: ${theme.shape.borderRadius};
+    font-size: ${theme.typography.body1.fontSize};
+    color: ${theme.palette.text.primary};
+    min-width: 150px;
+    :active {
+      background-color: ${theme.palette.button.light};
+    }
+    &.MuiButton-outlinedButton:hover {
+      background-color: ${theme.palette.button.main};
+    }
+    &.MuiButton-outlinedButton:active {
+      background-color: ${theme.palette.button.dark};
+      border-color: ${theme.palette.button.dark};
+    }
+    &.Mui-disabled:disabled,
+    &.Mui-disabled[disabled] {
+      background-color: ${theme.palette.secondary.light};
+      color: ${theme.palette.text.primary};
+    }
+
+    &.MuiButton-text {
+      color: ${theme.palette.button.main};
+    }
+    &.MuiButton-text:hover {
+      color: ${theme.palette.button.dark};
+      background-color: transparent;
+    }
+    &.MuiButton-text:active {
+      color: ${theme.palette.button.light};
```

Comment:
You might have to add the transparent background here as well. That's an edge case, but if you press on the button then slide your cursor outside the button the blue background shows:
<img width="207" alt="Screenshot 2022-03-14 at 16 00 57" src="https://user-images.githubusercontent.com/44675210/158211895-48bab447-d260-49fd-ac25-3293d3665279.png">

Reply:
Nice catch